install/roles/init/tasks/validate_params.yml

```diff
@@ -46,3 +46,8 @@
   when:
     - is_rhel_centos_6|bool
     - not use_docker|bool
+
+- name: Create tools installation directory
```

Comment:
Maybe change this to say `Create the Quipucords installation directory`

Reply:
Good catch! Totally missed this after the rename.